src/MicrosoftTeams.ts

```diff
@@ -55,6 +55,7 @@ namespace microsoftTeams {
 
     export interface TabInstance {
         tabName: string;
+        tabId: string;
```

Comment:
Hmm, why are we exposing this? Not a good idea; this is an internal detail of Teams and this ID is subject to mutation which is not made obvious here

Reply:
To navigate to a channel we need the channelId and the tabid. We can try to find the tabid when the method is invoked based on the tab properties passed and rely that there is only a tab with that name. 
Or we could also rename this to tabInstanceId

Reply:
As per our offline conversation we will indeed need this; however, I don't think we should expose it to the developers so it doesn't need to be on the interface.